docs/topics/mta-7-installing-web-console-on-ocp-local.adoc

```diff
@@ -0,0 +1,285 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-7-installing-web-console-on-ocp-local_{context}"]
+= Installing and configuring the {ProductName} Operator in a {rh-ocp-local} environment
+
+{rh-ocp-local} provides a quick and easy way to set up a local OpenShift cluster on your desktop or laptop. This local cluster allows you to test your applications and configuration parameters before sending them to production.
```

Comment:
We don't have an attribute for "OpenShift" in the MTA repo, we might want to make one.